opencontrail/map.jinja

```diff
@@ -1,96 +1,207 @@
+{%- set vendor = salt['pillar.get']('opencontrail:common:vendor', 'opencontrail') %}
 
-{% set common = salt['grains.filter_by']({
-    'Debian': {
-        'source': {'engine': 'pkg', 'address': 'http://'},
-        'pkgs': ['dpkg-dev','contrail-utils', 'contrail-nodemgr', 'gettext-base'],
-    },
-    'RedHat': {
-        'source': {'engine': 'pkg', 'address': 'http://'},
-        'pkgs': ['contrail-utils', 'contrail-nodemgr'],
-    },
-}, merge=pillar.opencontrail.get('common', {})) %}
+{%- load_yaml as base_defaults %}
+{%- if vendor in ['opencontrail'] %}
+common:
+  Debian:
+    vendor: opencontrail
+    source: {'engine': 'pkg', 'address': 'http://'}
+    pkgs:
+         ['dpkg-dev', 'contrail-utils', 'contrail-nodemgr', 'gettext-base', 'python-contrail']
+  RedHat:
+    vendor: opencontrail
+    source: {'engine': 'pkg', 'address': 'http://'}
+    pkgs:
+        ['contrail-utils', 'contrail-nodemgr']
+collector:
+  Debian:
+    pkgs:
+        ['contrail-analytics', 'python-cassandra']
+    redis_config: '/etc/redis/redis.conf'
+    services:
+        ['supervisor-analytics', 'redis-server']
+  RedHat:
+    pkgs:
+        ['contrail-analytics']
+    redis_config: '/etc/redis.conf'
+    services:
+        ['supervisor-analytics', 'redis']
+compute:
+  Debian:
+    pkgs:
+        ['contrail-vrouter-dkms', 'iproute2', 'haproxy']
```

Comment:
@pupapaik - see the other comments related to removal of contrail-vrouter virtual package.